robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -198,6 +198,7 @@ public boolean shouldAcquire(String name) {
             || name.startsWith("org.xml.")
             || name.startsWith("org.junit")
             || name.startsWith("org.hamcrest")
+            || name.startsWith("org.scalatest")
```

Comment:
Nope, `doNotAcquireClass` compares full class name: `classesToNotAquire.contains(name)`. I need to disable whole package (too many classes to add them one by one).
I would also prefer to add wildcard excludes on our side, but it's impossible with current api.